202009.0/ffa6653a-7d0e-41cc-a24e-074752b2ce1e.md

```diff
@@ -17,24 +13,20 @@ For Docker installation instructions, see one of the following:
 * [Installing Docker prerequisites on Linux](https://documentation.spryker.com/docs/installing-docker-prerequisites-on-linux)
 * [Installing Docker prerequisites on Windows](https://documentation.spryker.com/docs/installing-docker-prerequisites-on-windows)
 
-### Setting up a project with the Docker SDK
+### Adding Docker SDK to Spryker Project
 
 To set up a local project with the Docker SDK:
 
-1. Create the project directory and clone the source:
-```bash
-mkdir {project-name} && cd {project-name}
-git clone https://github.com/{project-url} ./
-```
+1. Using a terminal navigate to your project root folder, there you can find src/ directory and composer.json file.
 
-2. Clone the latest version of the Docker SDK:
+2. Clone the latest version of the Docker SDK into the project root folder:
```

Comment:
@sprymiker we need to place a correct docker.yaml to the root, 
if I didn't have any Docker SDK before. 

What template should I use?
How does the template correlate to the version of Docker?

Reply:
If you use demoshop boilerplate, you already have a bunch of deploy.yml files inside. If you did not use it for more than 2 years you will need an integration guide anyway.

Also one template does cover only one case, therefore we probably need multiple templates. So I assume we can just direct users to our demoshops boilerplates.